### PR TITLE
fix links

### PR DESCRIPTION
**Pull request by yxshv**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const SIDEBAR = {
     { text: "Installation", link: "en/installation" },
     { text: "Using Cakes", link: "en/using-cakes" },
     { text: "Creating Cakes", link: "en/creating-cakes" },
-    { text: "Advanced Usage", link: "en/advanced-usage" },
+    { text: "Advance Usage", link: "en/advance-usage" },
```
Comment (nexxeln) [inline src/config.ts]:
```ts
{ text: "Advanced Usage", link: "en/advanced-usage" },
```

File: src/pages/en/advance-usage.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Advanced Usage
-description: Cakecutter | Advanced Usage
+title: Advance Usage
```
Comment (nexxeln) [inline src/pages/en/advance-usage.md]:
```md
- title: Advanced Usage
```

File: src/pages/en/advance-usage.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+description: Cakecutter | Advance Usage
```
Comment (nexxeln) [inline src/pages/en/advance-usage.md]:
```md
- description: Cakecutter | Advanced Usage
```

**Review by nexxeln — CHANGES_REQUESTED** [review]
change to advanced usage everywhere

**Comment by yxshv** [comment]
done

**Comment by Dhravya** [comment]
@nexxeln you can merge after reviewing this one

**Pull request merged**